react/src/lib/components/DeckGLMap/components/ColorLegend.tsx

```diff
@@ -26,12 +26,14 @@ interface ColorLegendProps {
     horizontal?: boolean | null;
     layer: ExtendedLayer<unknown>;
     colorTables: colorTablesArray | string | undefined;
+    reverseRange?: boolean | null;
```

Comment:
Given the `?` this can be `bool` or `undefined`, so I don't think `null` is needed also.

Reply:
Agree. It was throwing lint error earlier so null was given but for latest change null is not required.